File: cpp_e9zMZ/cpp/03-vonglap/54.cpp
#include <iostream>
using namespace std;

int main() {
    int yourNumber = 279;
    int rightDigit = yourNumber % 10;
    yourNumber = yourNumber / 10;
    string result = "Yes";

    while (yourNumber > 0) {
        int currentDigit = yourNumber % 10;
        if (rightDigit <= currentDigit) {
            result = "No";
            break;
        }
        rightDigit = currentDigit;
        yourNumber = yourNumber / 10;
    }

    cout << result << endl;
}

File: python_ToSXs/python/03-vonglap/54.py
yourNumber = 123

result = "Yes"

rightDigit = yourNumber % 10
yourNumber = int(yourNumber / 10)

while yourNumber > 0:
    currentDigit = yourNumber % 10
    yourNumber = int(yourNumber / 10)

    if currentDigit >= rightDigit:
        result = "No"
        break

    rightDigit = currentDigit

print(result)
